nixos/modules/services/misc/paperless.nix

```diff
@@ -200,7 +199,17 @@ in
       description = lib.mdDoc "User under which Paperless runs.";
     };
 
-    package = mkPackageOption pkgs "paperless-ngx" { };
+    package = mkPackageOption pkgs "paperless-ngx" { } // {
+      apply = pkg: pkg.override {
+        tesseract5 = pkg.tesseract5.override {
+          # always enable detection modules
+          enableLanguages = if cfg.settings ? PAPERLESS_OCR_LANGUAGE then
+            [ "equ" "osd" ]
+              ++ lib.splitString "+" cfg.settings.PAPERLESS_OCR_LANGUAGE
+          else null;
+        };
+      };
+    };
```

Comment:
This breaks overridePythonAttrs used on the package.

Reply:
What should then be used instead? When I last looked at this, it wasn't easily possible to do an override in the package for the python package set and here.

Reply:
It could be an override on the default package, that users need to carry over.

Reply:
Since the override is a bit more complicated, I am not sure that this is the best option.